src/g3log/logworker.hpp

```diff
@@ -101,6 +102,12 @@ namespace g3 {
          return std2::make_unique<SinkHandle<T>> (sink);
       }
 
+      /// Removes a sink
+      /// @param
+      template<typename T>
+      void removeSink(std::unique_ptr<SinkHandle<T>> sink_handle) {

```

Comment:
That's correct. addSink returns a handle as a unique_ptr, which means caller is nominated with ownership. By removing the sink caller is giving up ownership and sink can be destroyed.